code/modules/events/wallrot.dm

```diff
@@ -1,10 +1,3 @@
-/datum/event/wallrot/setup()
```

Comment:
I would not remove the annoucement i would reword it. https://github.com/ParadiseSS13/Paradise/blob/b151dfb0b9b5b53e41a5a2605d1f38471a8e87df/code/game/turfs/simulated/walls.dm#L112

Wallrot weakens the damage walls can take before you can break it. so yeah it can be an issue.
Rather then biohazard perhaps something about station structural integrity?

Reply:
Do people actively hunt for wallrot after hearing the announcement? As far as I'm aware, everyone ignores it and will only (sometimes) deal with it if they organically come across it.

Reply:
> Do people actively hunt for wallrot after hearing the announcement? As far as I'm aware, everyone ignores it and will only (sometimes) deal with it if they organically come across it.

people don't for the most part

Reply:
if chem needs fungus for something. but REGARDLESS i think a rewording would be preferable to an outright removal

Reply:
> Do people actively hunt for wallrot after hearing the announcement? As far as I'm aware, everyone ignores it and will only (sometimes) deal with it if they organically come across it.

It gets dealt with but never on purpose from what I've seen.